packages/developer-portal/src/utils/launch-app.ts

```diff
@@ -1,9 +1,10 @@
 import { AppSummaryModel } from '@reapit/foundations-ts-definitions'
+import { reapitConnectBrowserSession } from '@/core/connect-session'
 
 export const handleLaunchApp = (app: AppSummaryModel) => {
-  import('../core/store').then(store => {
+  import('../core/store').then(() => {
     if (app.launchUri && app.id) {
-      if (store?.default?.state?.auth?.refreshSession?.mode === 'DESKTOP') {
+      if (reapitConnectBrowserSession.connectIsDesktop) {
```

Comment:
I have added a convenience method to the lib called `connectHasSession` which returns a boolean if there is a valid access token - might be useful here